python/py-pyside2/Portfile

```diff
@@ -103,6 +102,12 @@ if {${name} ne ${subport}} {
             qtremoteobjects
     }
 
+    # qt5-qtwebengine is currently broken on arm64, see https://trac.macports.org/ticket/62059
+    if { ${os.arch} ne "arm" && ${os.arch} ne "arm64" } {
+        qt5.depends_component \
+            qtwebengine
+    }
+
```

Comment:
`qt5-qtwebengine` is also broken on Monterey as well (x86-64 and arm64: https://trac.macports.org/ticket/63725), so that would seem like another case to disable here (if disabling at all is what should be done). With #12595 there could actually be a greater chance of `qt5-qtwebengine` working soon on Big Sur arm64 than at all on Monterey.

Reply:
I will happily add this check if the disabling is deemed desirable.

Reply:
Check added.